docs/editor/accessibility.md

```diff
@@ -158,7 +158,9 @@ Certain features such as folding and minimap (code overview) are disabled when i
 
 ## Terminal accessibility
 
-Output in the Integrated Terminal can be navigated through by using the "navigation mode" commands available in the Command Palette (press `kbstyle(F1)` and search for "terminal navigation mode").
+Terminal accessibility help can be toggled via `kbstyle(Alt+F1)` which describes some useful tips when using a screen reader. One of the tips described is to use `kbstyle(Shift+Tab)` to access the terminal's buffer, this will enter the screen reader's browse mode automatically (depending on your screen reader) and provide an accessible view of the entire terminal's buffer.
```

Comment:
```suggestion
Terminal accessibility help can be accessed via `kbstyle(Alt+F1)` which describes some useful tips when using a screen reader. One of the tips described is to use `kbstyle(Shift+Tab)` to access the terminal's buffer, this will enter the screen reader's browse mode automatically (depending on your screen reader) and provide an accessible view of the entire terminal's buffer.
```